backend/src/main/java/com/fake_orgasm/generator/flight_history_generator/Priority.java

```diff
@@ -0,0 +1,22 @@
+package com.fake_orgasm.generator.flight_history_generator;
+
+public enum Priority {
+    NORMAL("priority normal"),
```

Comment:
This class is repeated, because the user already has a category that is the same as this class, and also the categories (Priorities in this case) we are managing are VIP, Frequent, and Regular.

Reply:
The review has been solved